CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.15.1-rc.1]
```

Comment:
changelogs should not have `rc`s, here we should put `0.15.1`, since it will be merged to master once we release the final candidate.